### PR TITLE
Drop the timezone delta in the ISO8601 time parser

### PR DESCRIPTION
**Pull request by tobim**
###  :notebook_with_decorative_cover: Description

Up until now, we shifted time values according to the timezone modifier in the input. This is suboptimal because we're normally interested in normalized timestamps, and UTC is the natural representation for that.

###  :memo: Checklist

- [x] All user-facing changes have changelog entries.
- [ ] The changes are reflected on [docs.tenzir.com/vast](https://docs.tenzir.com/vast), if necessary.
- [x] The PR description contains instructions for the reviewer, if necessary.

### :dart: Review Instructions

Compare the change of the integration test expectations with the original data from the inputs.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,12 @@ This changelog documents all notable user-facing changes of VAST.
 
 ## Unreleased
 
+- ⚠️ The parser for ISO8601 timestamps now ignores timezone offsets. The old
+  behavior of shifting the timestamp according to the offset was problematic,
+  because it was not possible to know what zone the timestamp was relative to.
+  From now on all timestamps in VAST are in the UTC timezone.
+  [#1535](https://github.com/tenzir/vast/pull/1535)
```
Comment (lava) [inline CHANGELOG.md:20]: This seems a bit problematic: a time stamp like `15:30+01:00` unambiguously refers to 14:30 UTC, not 15:30 UTC.

So either I'm misunderstanding the changelog entry, or it looks like we'd be breaking ISO8601 compatibility with this change?

Comment (dominiklohmann) [reply CHANGELOG.md:20]: I don't understand the change either. Isn't the mismatch on the export side rather than on the import side?

Comment (tobim) [reply CHANGELOG.md:20]: @lava thanks for pointing this out. I must have misinterpreted the ISO definition.

Comment (tobim) [reply CHANGELOG.md:20]: I guess I can close this then :sweat_smile:

**Pull request closed without merging**